pravega/pravega
# Issue 6899: Update gradle-mkdocs-plugin to version 2.4.0

**Change log description**  
Update gradle-mkdocs-plugin to latest version so mkdocs doesn't fail

**Purpose of the change**  
Fixes #6899

**What the code does**  
Updates gradle-mkdocs-plugin to 2.4.0

**How to verify it**  
Run `./gradlew -PenableMkdocs=true mkdocsBuild` and see it succeed.

This change is incorporated into this prototype PR related to building and uploading mkdocs, where it can be seen to be working: https://github.com/derekm/pravega/pull/5
